app/assets/stylesheets/components/_markdown-editor.scss

```diff
@@ -1,30 +1,50 @@
-.app-c-markdown-editor__preview-toggle {
+.app-c-markdown-editor__head {
   @include govuk-font($size: 19);
 
   display: none;
+  position: relative;
+  z-index: 20;
```

Comment:
This causes the focus outline to be hidden and looks a bit weird - is that intentional?

Reply:
Also, the outline for the toggle buttons is slightly misaligned with the button borders.

Reply:
It is actually intentional, to position the toolbar over the textarea. I'll try to think of ways of enhancing it.